### ISSUE TITLE
Problem with api/v2/import-scan

### ISSUE THREAD

**Issue by altikamel** [DefectDojo/django-DefectDojo#11339]
Hello

I have an issue with using api/v2/import-scan from one pod (gitlab-runner) to defectdojo django pod in the same namespace.
I have installation of defect-dojo from helm chart, added my domain values to ingress to access the web interface and everything was successfull but i can't load my reports in json using API using service inside kubernetes namespace, but loading files from remote machine via ingress name is successful.

For example my service name is defectdojo-django, and i added it to DD_ALLOWED_HOSTS, so trying to use this service via cURL from inside of a gitlab runner pod is successful,connection successfully established, but when i try to use cURL to load my report json file with the service_name (not ingress name)  i have an errors such :
 expecting value: line 1 column 1 , bad request
  Bad Request - /api/v2/import-scan

when i try to load this file with all the same parameters via cURL from remote machine, using ingress, loading is successfull.

Also other methods of API work properly when i use cURL from gitlab runner pod via service inside namespace, only /apy/v2/import-scan doesnt work.

Does it mean that i should use only ingress name to load the peport to defectdojo , not service?

Thanks!

**Comment by testaccount90009** [comment]
Have you tried reimport-scan instead of import-scan?  Assuming you are using the same scan, of course.. Import =/= Reimport, but Reimport = Import.

**Comment by altikamel** [comment]
> Have you tried reimport-scan instead of import-scan? Assuming you are using the same scan, of course.. Import =/= Reimport, but Reimport = Import.

i tried it , reimport-scan returns same error - 

dojo.importers.base_importer - Expecting value: line 1 column 1 (char 0)
django.request - Bad request - api/v2/reimport-scan

**Comment by testaccount90009** [comment]
I can confirm import-scan and reimport-scan does work.  I'm using them as we speak for other tests I'm running on my Dev Dojo stack.  I don't use Helm, so it sounds like you have something else going on.  If it works with the ingress name and not service name, I'd use ingress name.

**Comment by altikamel** [comment]
> I can confirm import-scan and reimport-scan does work. I'm using them as we speak for other tests I'm running on my Dev Dojo stack. I don't use Helm, so it sounds like you have something else going on. If it works with the ingress name and not service name, I'd use ingress name.

Yes, my problem depends on restrictions of my k8s cluster when i cant use ingress inside the cluster between pods in the same namespace, only services..

**Comment by valentijnscholten** [comment]
The error implies an error with the contents/format of the report. Can you add more from the logs?
